cli/cli-client/src/main/java/org/infinispan/cli/logging/Messages.java

```diff
@@ -92,4 +95,64 @@
 
    @Message("Invalid resource '%s'")
    IllegalArgumentException invalidResource(String name);
+
+   @Message("No patches installed")
+   String patchNoPatchesInstalled();
+
+   @Message("%s")
+   String patchInfo(PatchInfo patchInfo);
+
+   @Message("The supplied patch cannot be applied to %s %s")
+   IllegalStateException patchCannotApply(String brandName, String version);
```

Comment:
I'm wondering if most of these methods should create a `CommandException` instead of `IllegalStateException` since we have descriptive messages... WDYT? 

The exception class doesn't bring anything useful in this case:
```
[disconnected]> patch rollback 
java.lang.IllegalStateException: No installed patches to roll back
```

Reply:
I wanted to keep the PatchTool class clean of aesh classes